cluster/gce/config-test.sh

```diff
@@ -179,7 +179,7 @@ ENABLE_METADATA_AGENT="${KUBE_ENABLE_METADATA_AGENT:-none}"
 
 # One special node out of NUM_NODES would be created of this type if specified.
 # Useful for scheduling heapster in large clusters with nodes of small size.
-HEAPSTER_MACHINE_TYPE="${HEAPSTER_MACHINE_TYPE:-}"
+HEAPSTER_MACHINE_TYPE="${HEAPSTER_MACHINE_TYPE:-n1-standard-4}"
```

Comment:
We don't want this in my opinion.
This file is used for every single test and we don't want to have 4-core heapster machine in bazillions of different test suites we are running.

Reply:
Sure, I just put this here to trigger the logic in the gce-100-performance presubmit and validate the change. I'm going to remove this before submitting.

Reply:
it worked, removing.